server/src/database/scripts/sync-chefs-data.service.ts

```diff
@@ -0,0 +1,88 @@
+import axios from 'axios';
+import { Injectable } from '@nestjs/common';
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Application } from '../../application/application.entity';
+import { REQUEST_METHODS } from '../../common/constants';
+import { ApplicationService } from '@/application/application.service';
+import { SaveApplicationDto } from '@/common/dto/save-application.dto';
+import { ReviewStatuses } from '@/common/enums';
+
+// CHEFS Constants
+const CHEFS_FORM_IDS = ['4b19eee6-f42d-481f-8279-cbc28ab68cf0'];
+const CHEFS_BASE_URL = 'https://submit.digital.gov.bc.ca/app/api/v1';
+
+@Injectable()
+export class SyncChefsDataService {
+  constructor(
+    @InjectRepository(Application)
+    private readonly applicationRepo: Repository<Application>,
+    private readonly appService: ApplicationService = new ApplicationService(applicationRepo)
```

Comment:
private readonly appService: ApplicationService will itself inject the service object. No need of new!